cloudinit/config/cc_growpart.py

```diff
@@ -210,11 +210,35 @@ def get_size(filename):
 
 def device_part_info(devpath):
     # convert an entry in /dev/ to parent disk and partition number
+    myenv = os.environ.copy()
+    myenv['LANG'] = 'C'
 
     # input of /dev/vdb or /dev/disk/by-label/foo
     # rpath is hopefully a real-ish path in /dev (vda, sdb..)
     rpath = os.path.realpath(devpath)
 
+    is_lvm = True if re.search('.*dm.*', rpath) else False
```

Comment:
i still think this should be a standalone function
that way we

- can lru cache it
- call it, instead of passing it around
- check for is_Linux(), which is also cached
- (check foris_AIX(), the only other platform with LVM support, as soon as someone adds AIX support to cloud-init)

Reply:
Oh yeah, forgot to address this comment from your last review. Sorry! I think we could do it, even though the optimization would not be so big. And even though this is a tiny one liner :-) But if you guys like the idea I can change it, no problem for me.